dev-tools/mage/gotest.go

```diff
@@ -329,117 +317,10 @@ func makeCommand(ctx context.Context, env map[string]string, cmd string, args ..
 	c.Stderr = os.Stderr
 	c.Stdin = os.Stdin
 	log.Println("exec:", cmd, strings.Join(args, " "))
+	fmt.Println("exec:", cmd, strings.Join(args, " "))
 	return c
 }
 
-// GoTestSummary is a summary of test results.
-type GoTestSummary struct {
```

Comment:
GoTestSummary is not used anymore. `gotestsum` prints it's own summary, not as rich as this one, but should be good enough.